### DISCUSSION TITLE
Unsupported video mode for SNES games

### DISCUSSION THREAD

**Discussion by mojicakes** [eduardofilo/RG350_adam_image#240] [RetroArch topics]
All of the snes games that I've tried has this problem where it shows the unsupported video mode error after loading the game. I tried changing the emulators (theres like 3 of them) but to no avail it still shows the error.

**Comment by eduardofilo** [comment]
What is your device model?

**Comment by mojicakes** [comment]
> What is your device model?

anbernic rg280v

**Comment by eduardofilo** [comment]
That is weird because it is precisely the device I use to develop/test the image. I can assure you that I have no problems with SNES games on RG280V. Tell me one in particular to try to test it (indicating if it is the PAL or NTSC version in case that is the reason).

**Comment by mojicakes** [comment]
I manually switched it first to Pal and the screen is just all black. Then back to NTSC and the error showed up. 
![278507128_1407174766370374_2067807116054303715_n](https://user-images.githubusercontent.com/104123807/164489688-19aa9e9b-1232-43f3-9b96-9a02c79c4125.jpg)

I tried playing the games(Final Fantasy 2, F-Zero, Super Metroid, Etc.) on another device(Retroid Pocket 2+) and it ran fine

**Reply by eduardofilo** [reply]
I meant that you should indicate whether the version of the games you were testing was PAL or NTSC (this is usually indicated in brackets in the ROM names).
I've tried the USA (then NTSC) version of F-Zero for example and it plays without problems on a freshly installed Adam 1.4.2. If you haven't modified the RetroArch configuration, then there is no other option than your ROMs have some kind of modification.

![IMG_20220421_202408](https://user-images.githubusercontent.com/2923728/164528607-7b9f0c14-6935-4998-a5dd-5f1b8de2de0d.jpg)

**Comment by mojicakes** [comment]
Oh I misunderstood that. I didn't change anything in the settings cause I don't know anything about it. I tried USA and Europe Roms and the error still shows up after loading the game. For some reason I manage to run a single game Dual Orb II (English Patched - v1.7). It is fully playable and doesn't crash or cause error.
